Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -455,35 +455,20 @@ - (void)_updateScreenTimeViewGeometry
     [[_screenTimeWebpageController view] setFrame:bounds];
 }
 
-- (void)_updateScreenTimeShieldVisibilityForWindow
+- (void)_updateScreenTimeBasedOnWindowVisibility
 {
     BOOL viewIsInWindow = !!self.window;
-
-    BOOL showsSystemScreenTimeBlockingView = [_configuration _showsSystemScreenTimeBlockingView];
-
-    if (viewIsInWindow) {
-        if (!showsSystemScreenTimeBlockingView && _screenTimeBlurredSnapshot) {
-            [_screenTimeBlurredSnapshot setHidden:NO];
-            RELEASE_LOG(ScreenTime, "Screen Time has updated visibility to show blurred view.");
-        } else if (showsSystemScreenTimeBlockingView) {
-            [[_screenTimeWebpageController view] setHidden:NO];
-            RELEASE_LOG(ScreenTime, "Screen Time has updated visibility to show system shield.");
-        }
-    } else {
-        if (_screenTimeBlurredSnapshot) {
-            [_screenTimeBlurredSnapshot setHidden:YES];
```

Comment:
What updates the blur view now?

Reply:
Realized that this was never necessary, since if the view is not in window, it wouldn't be visible anyway and neither would the blur.